datahub-web-react/src/app/entity/Access/Access.tsx

```diff
@@ -0,0 +1,103 @@
+import { TagOutlined, TagFilled } from '@ant-design/icons';
+import * as React from 'react';
+import styled from 'styled-components';
+import { Role, EntityType, SearchResult } from '../../../types.generated';
+import DefaultPreviewCard from '../../preview/DefaultPreviewCard';
+import { Entity, EntityCapabilityType, IconStyleType, PreviewType } from '../Entity';
+import { getDataForEntityType } from '../shared/containers/profile/utils';
+import { urlEncodeUrn } from '../shared/utils';
+import { SidebarAboutSection } from '../shared/containers/profile/sidebar/AboutSection/SidebarAboutSection';
+import { useGetExternalRoleQuery } from '../../../graphql/accessrole.generated';
+import { EntityProfile } from '../shared/containers/profile/EntityProfile';
+
+const PreviewTagIcon = styled(TagOutlined)`
+    font-size: 20px;
+`;
+// /**
+//  * Definition of the DataHub Access Role entity.
+//  */
+export class RoleEntity implements Entity<Role> {
```

Comment:
please name this file RoleEntity.tsx to be consistent with other similar files :)

Reply:
done update to RoleEntity.tsx